ush/fire_emiss_tools.py

```diff
@@ -103,10 +135,34 @@ def estimate_fire_duration(intp_avail_hours, intp_dir, fcst_dates, current_day,
     return(te)
 
 def save_fire_dur(cols, rows, te):
+    """
+    ???
+
+    cols: ???
+    rows: ???
+    te: ???
+    
+    """
     fire_dur = np.array(te).reshape(cols, rows)
     return(fire_dur)
 
 def produce_emiss_file(xarr_hwp, frp_avg_reshaped, totprcp_ave_arr, xarr_totprcp, intp_dir, current_day, tgt_latt, tgt_lont, ebb_tot_reshaped, fire_age, cols, rows):
+    """
+    ???
+
+    xarr_hwp: ???
+    frp_avg_reshaped: ???
+    totprcp_ave_arr: ???
+    xarr_totprcp: ???
+    intp_dir: ???
+    current_day: ???
+    tgt_latt: ???
+    tgt_lont: ???
+    ebb_tot_reshaped: ???
+    fire_age: ???
+    cols: ???
+    rows: ???
+    """

```

Comment:
@haiqinli and @MatthewPyle-NOAA what do each of these parameters mean and what does this function do?